python/src/llvm.cc

```diff
@@ -302,6 +303,21 @@ void init_triton_llvm(py::module &&m) {
         PassBuilder pb(nullptr /*targetMachine*/, tuningOptions, std::nullopt,
                        instrCbPtr);
 
+        std::string pluginFile =
+            mlir::triton::tools::getStrEnv("AMDGCN_INSTRUMENTATION_LIB");
+        if (!pluginFile.empty()) {
+          llvm::errs()
+              << "Adding AMDGCN instrumentation pass to Triton pipeline"
+              << "\n";
```

Comment:
As far as I know, we do not have any logging policy at the moment.
In general Triton prints messages only if something broken, which is not the case here.

I think it is better to comment this print with TODO and leave logging part for later to speed up merge of this particular change.

Reply:
Changed to dbgs

Reply:
Ended up just pulling it out and added the TODO like you suggested and we can figure out later.